warp10/src/main/java/io/warp10/WarpConfig.java

```diff
@@ -326,10 +326,11 @@ public static Properties readConfig(boolean exitOnError, Reader reader, Properti
       }
 
       //
-      // Ignore empty properties
+      // Remove property when empty
```

Comment:
This should be donne on line 313, with a check on the value returned by the call to remove to emit one of the two following warnings:

Reply:
LOG.warn("Empty value for property '" + tokens[0] + "' on line " + lineno + ", ignoring."); if the property was not defined annd LOG.warn("Empty value for property '" + tokens[0] + "' on line " + lineno + ", has cleared the previous value of '" + oldvalue + "'");

Reply:
and this must be done after url decode and trim.  done.